apps/dh/api-dh/source/DataHub.WebApi/GraphQL/GraphQLQuery.cs

```diff
@@ -134,8 +138,8 @@ public GraphQLQuery()
 
             Field<NonNullGraphType<ListGraphType<NonNullGraphType<BatchType>>>>("batches")
                 .Argument<DateRangeType>("executionTime")
-                .Argument<BatchState>("executionState", nullable: true)
-                .Argument<ProcessType>("processType", nullable: true)
+                .Argument<BatchState>("executionState", true)
```

Comment:
Suggestion: keep named param, so it is easier to see what is set to true

Reply:
fixed